ibis/backends/tests/test_array.py

```diff
@@ -671,6 +684,14 @@ def test_array_remove(con, input, expected):
             {"a": [[1, 3, 3], [], [42, 42], [], [None], None]},
             [{3, 1}, set(), {42}, set(), {None}, None],
             id="null",
+            marks=[
+                pytest.mark.notyet(
+                    ["pyspark"],
+                    condition=IS_SPARK_REMOTE,
+                    raises=AssertionError,
+                    reason="somehow, transformed results are different types",
```

Comment:
I believe these are all the result of Spark connect unconditionally using Arrow for transport, which is great long term, but not compatible with a bunch of our existing array tests that want Python `None`s and not `numpy.nan`s.